### PR TITLE
Analytics fixes and style updates.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
## Proposed changes

All analytics errors and style errors fixed.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- None.

## Special notes for your reviewer:
Bug fix PR.

### CODE REVIEW

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/PopOver/styles.ts [hunk 1]
```diff
@@ -51,13 +51,10 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-    width: 20,
-    height: 22,
-  },
-  dateRangeIcon: {
-    color: theme.palette.secondary.contrastText,
-    width: 20,
+    width: 23,
     height: 22,
+    paddingLeft: 3,
+    marginTop: 2,
```
Comment (oumkale) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/PopOver/styles.ts:57]: Use Theme for style use.

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseWorkflow/index.tsx [hunk 2]
```diff
@@ -295,7 +295,7 @@ const BrowseWorkflow = () => {
                 {/* Workflow Name */}
                 <TableCell className={classes.workflowName}>
                   <div style={{ display: 'flex', flexDirection: 'row' }}>
-                    <Typography style={{ paddingTop: 10 }}>
+                    <Typography style={{ paddingTop: 12 }}>
```
Comment (oumkale) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseWorkflow/index.tsx:298]: Use Theme for style use.

**Comment by ishangupta-ds** [comment]
@oumkale PTAL.

**Review by amityt — APPROVED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Review by Jonsy13 — DISMISSED** [review]
LGTM

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]

**Review by Jonsy13 — DISMISSED** [review]

**Review by S-ayanide — DISMISSED** [review]

**Review by arkajyotiMukherjee — DISMISSED** [review]

**Review by Jonsy13 — DISMISSED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by oumkale — DISMISSED** [review]

**Review by gdsoumya — DISMISSED** [review]

**Review by S-ayanide — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 3]
```diff
@@ -95,7 +95,7 @@ workflowUnderground:
   heading: 'Click on test to see detailed log of your workflow'
 
 community:
-  heading: 'Daily Insights'
+  heading: 'Litmus Insights'
   headingDesc: 'Stats for the Litmus community in the last 24 hours'
```
Comment (S-ayanide) [inline litmus-portal/frontend/public/locales/en/translation.yaml:99]: Change this to 'Stats for the Litmus Community'

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Review by oumkale — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]

**Pull request merged**